docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -3,15 +3,14 @@
 Model Optimizer can convert all floating-point weights to `FP16` data type. The resulting IR is called
 compressed `FP16` model.
 
-To compress the model, use the `--data_type` option:
+Starting from 2022.3 release, option --data_type is deprecated, instead use --compress_to_fp16 in the following way:
+
+To compress the model, use the `--compress_to_fp16` option:
```

Comment:
Here we are describing FP16 compression feature. Please describe it _properly_. Provide information why user may want to apply this compression and how it will affect execution/IR size (move statements from the end of this document to the beginning). Then explain how it can be activated in MO. And then mention `--data_type` option which is deprecated in `**NOTE**` paragraph and provide a small transition guide for it like `instead of --data_type FP16 please use --compress_to_fp16.` Explain effect of `--data_type FP32` application and state that this is useless. Provide more words to make it more understandable.

Reply:
Done.